app/src/main/java/org/hackillinois/android/view/SplashScreenActivity.kt

```diff
@@ -0,0 +1,61 @@
+package org.hackillinois.android.view
+
+import android.content.Context
+import android.content.Intent
+import android.support.v7.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import org.hackillinois.android.App
+
+import org.hackillinois.android.R
+import org.hackillinois.android.model.User
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+
+class SplashScreenActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        var jwt = loadJWT();
+
+        if(jwt != "") {
```

Comment:
Kinda picky, but maybe we should make empty String a constant like `defaultJWT` and also put that in `loadJWT()` to make it clear what's happening here.